ui/app/routes/vault/cluster/settings/auth/enable.js

```diff
@@ -9,11 +9,6 @@ import { inject as service } from '@ember/service';
 export default class VaultClusterSettingsAuthEnableRoute extends Route {
   @service store;
 
-  beforeModel() {
-    // Unload to prevent naming collisions when we mount a new engine
```

Comment:
We're unloading dirty models when exiting `mount-backend-form` so if there's a naming collision that's not ok anyway